app/controllers/v1/leaders_controller.rb

```
@@ -45,5 +47,40 @@ def welcome_letter_for_leader(leader)
         address: leader.address
       )
     end
+
+    def welcome_message(leader)
+      token = access_token leader
+      return if token.nil?
```

Do we want this to fail silently?